content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,68 @@
+================================================================
+Electronic invoicing (:abbr:`EDI (electronic data interchange)`)
+================================================================
+
+EDI, or electronic data interchange, is the inter-company communication of business documents, such
+as purchase orders and invoices, in a standard format. Sending documents according to an EDI
+standard ensures that the machine receiving the message can interpret the information correctly.
+Various EDI file formats exist.
```

Comment:
Is it various formats according to countries? If so maybe it's nice to specify it, at this point of the page.